source/Applications/Advanced/Downsample/Downsample.cpp

```diff
@@ -86,11 +86,10 @@ Zivid::PointCloud downsample(const Zivid::PointCloud &pointCloud, int downsampli
     Eigen::MatrixXi greenDownsampled = downsampleAndRound(g, downsamplingFactor);
     Eigen::MatrixXi blueDownsampled = downsampleAndRound(b, downsamplingFactor);
 
-    std::function<float(float)> is_not_nan_functor = [](float f) { return !std::isnan(f); };
-    std::function<float(float)> nan_to_zero_functor = nanToZero;
+    std::function<float(float)> isNotNanFunctor = [](float f) { return !std::isnan(f); };
+    std::function<float(float)> nanToZeroFunctor = nanToZero;
```

Comment:
@knatten we've introduced the snake_case notation for functors. Are you ok with removing this? It seems clang-tidy is requesting it.

Reply:
What do you mean? That you have agreed on using snake_case for functors, but camelCase for all other variables?

What are you asking if I'm ok with removing?

What is clang-tidy requesting, and how?

Reply:
You helped me with these functors. But maybe I don't remember it right. It may be that what you suggested was to add 'Funtor' to the name, and nothing about using the snake_case. Maybe the snake_case we just didn't notice at the time. I just remember you were involved and there was a suggestion to do something differently...

I am asking if you are ok to change snake_case to camelCase.

I think clang-tidy is complaining about the snake_case, right @torbsorb ?

Reply:
It is just a variable as anything else, think this is fine.

Reply:
What @nedrebo said.